src/AudioIO.cpp

```diff
@@ -1447,10 +1450,31 @@ void AudioIO::StopStream()
       mPortStreamV19 = NULL;
    }
 
+
+
+   // We previously told AudioThread to stop processing, now let's
+   // be sure it has really stopped before resetting mpTransportState
+   while (!mAudioThreadStopAcknowledged.load())
+   {
+      using namespace std::chrono;
+      std::this_thread::sleep_for(50ms);
+   }
+   mAudioThreadStopAcknowledged.store(false);
```

Comment:
I removed the memory ordering specifications, because from what I read on cppreference.com, I understood that leaving the defaulted order  is actually equivalent to using release on store and acquire on load.

from https://en.cppreference.com/w/cpp/atomic/atomic/store

`void store( T desired, std::memory_order order = std::memory_order_seq_cst ) noexcept;`

from https://en.cppreference.com/w/cpp/atomic/memory_order

memory_order_seq_cst 	
"A load operation with this memory order performs an acquire operation, a store performs a release operation..."

Still, if we want to have more explicit code, I can specify the order

Reply:
It is not equivalent.  It implies stronger conditions that might generate different code, depending on the processor.  As I understand, it's not actually stronger for Intel, but now we are not targeting only Intel.

Reply:
alright, I went back to the original load-acquire + store-release scheme